src/themes/_dark.scss

```diff
@@ -0,0 +1,61 @@
+@use '~@angular/material/theming' as *;
+@use 'common/utility';
+@use '../app/core/header/header.component.theme' as header;
+
+
+$primary-colors: utility.add-colors($mat-gray, (
+  "white": white,
+  "black": black,
+  850: #323232,
+  light-header: #444A65,
```

Comment:
Need to provide contrasts for these as well or we'll might run into problems later